dev-python/uri_template/uri_template-1.1.0.ebuild

```
@@ -0,0 +1,27 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_HASH="8547d60a57dd8aabffa7063fbc0b120c9902a130"
+
+PYTHON_COMPAT=( pypy3 python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="URI Template expansion in strict adherence to RFC 6570"
+HOMEPAGE="https://gitlab.linss.com/open-source/uri_template/"
+SRC_URI="https://gitlab.linss.com/open-source/${PN}/-/archive/v${PV}/${P}.tar.gz"
```

I'm not too familiar with GitLab's conventions but I think the reason you've had to do the `MY_HASH` stuff is because this is referring to `uri_template-1.1.0.tar.gz` and not `uri_template-v1.1.0.tar.gz`.

If we instead do

```suggestion
SRC_URI="https://gitlab.linss.com/open-source/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.gz"
```

We can drop `MY_HASH` and do `S="${WORKDIR}"/${PN}-v${PV}`